pkg/log/log.go

```diff
@@ -81,7 +83,8 @@ var (
 // set to a NullLogSink.
 var (
 	dlog = NewDelegatingLogSink(NullLogSink{})
-	Log  = logr.New(dlog)
+	// Log uses `klog` as the controller-runtime default logger.
+	Log = klog.NewKlogr()
```

Comment:
This is a breaking change. Controller-Runtime doesn't set any logger by default. Please just set your desired logger at the start of your program.
/hold

Reply:
Got it, raised a new PR https://github.com/kubernetes-sigs/kubebuilder/pull/3033 to set the default logger.
PTAL thanks.